app/src/models/popup.ts

```diff
@@ -362,4 +363,13 @@ export type Popup =
     }
   | {
       type: PopupType.StartPullRequest
+      allBranches: ReadonlyArray<Branch>
+      currentBranch: Branch
+      defaultBranch: Branch | null
+      hideWhitespaceInHistoryDiff: boolean
+      imageDiffType: ImageDiffType
+      pullRequestState: IPullRequestState
```

Comment:
Next PR is going to end up removing pullRequestState but rest of variables are simply inputs so they will should remain on the popup model